### PR TITLE
New package: CABS.SalesPrix version 6.15.0.0

### PR DESCRIPTION
**Pull request by KlausGue**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115273&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: CABS.SalesPrix
+PackageVersion: 6.15.0.0
+PackageLocale: de-DE
+Publisher: CAB+S GmbH
+PackageName: SalesPrix
```
Comment (stephengillie) [inline manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml:8]:
```suggestion
PackageName: SalesPrix
PackageUrl: https://www.scanprix.de/
```

**Comment by KlausGue** [comment]
> Hello @KlausGue,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

Hello,
i don't understand. wingetcreate could download this file. If you enter the link, the brwoser start downloading. What is going wrong?

File: manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml (hunk 1 above)
Comment (KlausGue) [reply manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml:8]: Hello
i don't understand what you are saying. I did what wingetcreate tells me to do. The name of the program is "SalesPrix", the name of the company is "CAB+S GmbH", in short: CABS.
What is the difference between "PackageIdentifier" and "PackageName"?
Please explain.
Best regards
Klaus

Comment (stephengillie) [reply manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml:8]: Hi @KlausGue,

> i don't understand what you are saying. 

I'm suggesting to add a PackageUrl that points at the application's website. This helps satisfy the requirement that it's possible to navigate from the PackageUrl to the InstallerUrl. Sorry for being obtuse. 

> I did what wingetcreate tells me to do. 

This isn't a required field, so WingetCreate might not ask for it. 

> The name of the program is "SalesPrix", the name of the company is "CAB+S GmbH", in short: CABS.

Yes, the PackageIdentifier is fine, which is why I didn't suggest any changes to it. 

> What is the difference between "PackageIdentifier" and "PackageName"?

Here's the `defaultLocale` entry for our [1.4.0 Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/defaultLocale.md).
- PackageIdentifier is a unique way of identifying the package across the WinGet package manager's systems and components. 
- PackageName is used in displaying the package locally, and should match the Apps or Add and Remove Programs name. 

Does this help? The PR is well-formed and should merge without further changes.

**Comment by stephengillie** [comment]
> Hello, i don't understand. wingetcreate could download this file. If you enter the link, the brwoser start downloading. What is going wrong?

Our robot wasn't familiar with this domain, I'm guessing, and needed a human to verify that it's not a 3rd party download/mirror site. As we require packages to be hosted by their Independent Software Vendor (ISV, such as the publisher or developer). Sorry for any concern caused.

**Comment by KlausGue** [comment]
i added PackageUrl : https://www.scanprix.de
is this correct?

**Comment by stephengillie** [comment]
> i added PackageUrl : https://www.scanprix.de is this correct?

Yes, it's correct.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**